spacemacs/packages.el

```diff
@@ -136,6 +137,35 @@ which require an initialization must be listed explicitly in the list.")
       (setq ace-jump-mode-scope 'global)
       (evil-leader/set-key "`" 'ace-jump-mode-pop-mark))))
 
+(defun spacemacs/init-ace-link ()
+  (use-package ace-link
+    :defer t
+    :init (ace-link-setup-default)
```

Comment:
This line will defeat the lazy loading, it should be in `:config`.
Instead of `:defer t` you can try `:commands (spacemacs/ace-buffer-links ...)` the ellipsis means the stock interactive functions of ace-link.